notebooks/nx_image_demo.livemd

```diff
@@ -0,0 +1,146 @@
+# Nx.Image Demonstration
+
+```elixir
+Mix.install([
+  {:nx, "~> 0.4.0"},
+  {:kino_vega_lite, "~> 0.1.6"},
+  {:kino, "~> 0.7.0"},
+  {:nx_image, git: "https://github.com/elixir-nx/nx_image.git"},
+  {:stb_image, "~> 0.5.2"},
+  {:vega_lite, "~> 0.1.6"},
+  {:file_upload_smart_cell, git: "https://github.com/akoutmos/file_upload_smart_cell.git"}
+])
+```
+
+## Upload your test image
```

Comment:
The Kino main branch has a `Kino.Input.image/1`, which supports file uploads and, best of all, returns the image in RGB / Tensor format input. Could you please update this section to use that instead and remove the `file_upload_smart_cell` dependency for now? :)

https://github.com/livebook-dev/kino/blob/5a015e6b9f2097452180457d1c39277b4b893295/lib/kino/input.ex#LL262C17-L262C22

Reply:
I figured out that I needed to use the latest Livebook code.  I succeeded in adding the Kino.Input.image("label) to a notebook.  However, I'm having difficulty figuring out how to get any data about the image (size or tensor).  

img = Kino.Input.image("label")

IO.inspect(img)
-->
%Kino.Input{
  attrs: %{
    default: nil,
    destination: {Kino.SubscriptionManager, :"dceq47pv-livebook_iaakhca6@ml3"},
    fit: :contain,
    format: :rgb,
    id: "83729853",
    label: "label",
    ref: "l266kaqz5xj753jhfry5i2sp76cfdnsy",
    size: nil,
    type: :image
  }
}

Reply:
I figured out that I need to read the input.

Kino.Input.read(img)

Reply:
Switched to Kino.Input.image and removed StgImage.